images/nginx/rootfs/build.sh

```diff
@@ -14,14 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 set -o errexit
 set -o nounset
 set -o pipefail
 
 export DEBIAN_FRONTEND=noninteractive
 
-export OPENRESTY_VERSION=1.15.8.2
+export OPENRESTY_VERSION=1.15.8.1
```

Comment:
why are you downgrading?

Reply:
Because there's no 1.15.8.2 release in the openresty page.

Reply:
hrm, https://openresty.org/download/openresty-1.15.8.2.tar.gz is available though and should introduce the latest CVE patches

Reply:
Yes, but that tgz does not contains all the packages and when I tried to use it I got lots of errors.

Reply:
Also, I am waiting for a 1.17 release before continuing with this PR

Reply:
oh interesting, maybe @thibaultcha can comment on this

Reply:
The tarball is now available at https://openresty.org/download/openresty-1.15.8.2.tar.gz. It took some time since only agentzh can upload them to openresty.org at the moment, and he was travelling when 1.15.8.2 was released. It is the same as the GitHub release tarball I uploaded a few days ago already!

Reply:
@thibaultcha thanks for the update.